### ISSUE TITLE
Training on custom dataset

### ISSUE THREAD

**Issue by adityabobade95** [wentaozhu/AutoShot#5]
Hi, 
Thank you for sharing your amazing work. I have tried the trained model on my test cases and it works well for most of my test cases. I wanted to improve the model accuracy for my dataset and wanted to train the model. 
Can you please share the training script? Or Give the hyper-parameter that was used for model training?
Looking forward to your response.

Thanks in advance.

**Comment by lynshwoo2022** [comment]
have you solved this problem?

**Comment by manoj-samal200** [comment]
i also need the training script if you guys get it please do share the code..
thanks
